docs/advanced/shipping/custom-shipping-information.mdx

```diff
@@ -117,12 +117,13 @@ You can reference one of our implementations to get you started
 
 - [`<ModialRelay />`](https://gitlab.com/front-commerce/front-commerce/-/blob/main/modules/shipping-mondialrelay/web/theme/modules/MondialRelay/MondialRelay.js)
 - [`<Colissimo />`](https://gitlab.com/front-commerce/front-commerce/-/blob/main/modules/shipping-colissimo-magento2/web/theme/modules/Colissimo/Colissimo.js)
+- [`<ChronoRelais />`](https://gitlab.com/front-commerce/front-commerce/-/blob/main/modules/shipping-chronorelais/web/theme/modules/ChronoRelais/ChronoRelais.js)
```

Comment:
It should be
```suggestion
- [`<ChronoRelais />`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/modules/shipping-chronorelais/web/theme/modules/ChronoRelais/ChronoRelais.js)
```

We should merge `main` into `next` (to get changes from !502). I'll do it right away.